core/embed/upymod/modtrezorio/modtrezorio-ble.h

```diff
@@ -181,6 +185,12 @@ STATIC mp_obj_t mod_trezorio_BLE_disconnect(void) {
 STATIC MP_DEFINE_CONST_FUN_OBJ_0(mod_trezorio_BLE_disconnect_obj,
                                  mod_trezorio_BLE_disconnect);
 
+/// RX_PACKET_LEN: ClassVar[int]
+/// """Length of one BLE RX packet."""
+
+/// TX_PACKET_LEN: ClassVar[int]
+/// """Length of one BLE TX packet."""
+
```

Comment:
```suggestion
/// RX_PACKET_LEN: int
/// """Length of one BLE RX packet."""
/// TX_PACKET_LEN: int
/// """Length of one BLE TX packet."""
```
these should not be `ClassVar`; the reason HID's are a ClassVar is that there's `class HID` encapsulating them.

we prooooobably don't need to introduce a class here now; perhaps in the future if we expose the management of multiple connections